afids_utils/tests/test_afids.py

```diff
@@ -177,25 +176,15 @@ def test_valid_load(
         assert isinstance(afids_set.slicer_version, str)
         assert isinstance(afids_set.coord_system, str)
         assert isinstance(afids_set.afids, list)
-        assert isinstance(afids_set.afids[label], AfidPosition)
+        assert isinstance(afids_set.afids[label - 1], AfidPosition)
 
     def test_invalid_fpath(self):
         with pytest.raises(FileNotFoundError, match=".*does not exist"):
             AfidSet.load("invalid/fpath.fcsv")
 
-    @given(
-        ext=st.text(
-            min_size=2,
-            max_size=5,
-            alphabet=st.characters(
-                min_codepoint=ord("A"), max_codepoint=ord("z")
-            ),
-        )
-    )
-    @settings(
-        suppress_health_check=[HealthCheck.function_scoped_fixture],
-    )
-    def test_invalid_ext(self, valid_fcsv_file: PathLike[str], ext: str):
+    @given(ext=af_st.gen_chars())
+    @allow_function_scoped
```

Comment:
You may be right, this might be a remnant of files were being passed. A lot of these are originally introduced everytime the warning/error popped up when running the tests. I'll go through them and try to clean this up